api/src/verify.rs

```diff
@@ -89,7 +89,7 @@ pub async fn configured_openid4vp_mdl_request(
             GeneralName::UniformResourceIdentifier(uri) => Some(uri.to_string()),
             _ => None,
         })
-        .collect();
+        .next().ok_or(CustomError::InternalError("The reader has a certificate without a SubjectAltName while using client_id_scheme x509_san_uri".to_string()))?;
```

Comment:
Technically speaking they are missing a URI SAN

Reply:
Yes I was worried about that too, but it works.
I will update our reader signer cert though for accuracy